### ISSUE TITLE
Geocoding API 403 for 100%

### ISSUE THREAD

**Issue by raichea** [astuder/lightroom-map-fix#7]
As mentioned in the "No luck" issue, I am also seeing 100% Geocoding API failures (403 code returned).

Applying the patch was very straightforward and the maps are now appearing as expected. Every now and then I see a "Map request error" message pop up briefly in LR (6.14 on Win7 64-bit).

I've disabled both checkboxes for address lookup as I don't use this feature, but the error remains.

**Comment by raichea** [comment]
I have tried modifying "nature" in LOCATIONDEBUGPANEL.LUA but this doesn't change the behaviour.

I don't see a place name at top right (don't recall ever seeing that. but I don't use maps that often). All I see is an icon for expanding to fullscreen (and back) but apart from changing the icon doesn't do anything.

**Comment by astuder** [comment]
Do you have the Geocoding API enabled in your Google Cloud account? Did you patch AGREVERSEGEOCODESERVICE.LUA with your API key?

The location worked for me without the 'nature' hack (I don't use address lookup).

**Comment by raichea** [comment]
Ah! After I enabled Geocoding, I went back to the API restrictions list - it hadn't changed but the dropdown didn't include the Geocoding API so I couldn't add it. I assumed (wrongly) that Geocoding was handled differently. 

I've just gone back in there to check things and now Geocoding **is** now in the dropdown. I've added it to the restricted list and things now work as expected - I'm not getting the errors and the location is appearing top right. This is also true after I backed out the "nature" patch.

It seems that it takes a little while for Geocoding to propagate to the dropdown.

Fantastic piece of work... much appreciated.
